### DISCUSSION TITLE
Plugin without UI

### DISCUSSION THREAD

**Discussion by dhrblav** [molstar/molstar#480] [Q&A] [answered]
Is there a proper way to use Mol* without any UI?
I would like to benefit from a number of Mol* functions such as computing the accessible surface area of a structure, retrieving non-covalent bonds or disulfide bridges, without a 3D view.
Is this use case supported?

**Comment by dsehnal** [comment, marked as answer]
I don't know about "proper", but yes, you can create the representations of structures and manipulating them without creating a plugin instance. For example `src/servers/model` does this in a non-trivial way.
